src/sentry/grouping/enhancement-configs/legacy@2019-03-12.txt

```diff
@@ -24,10 +24,12 @@ family:native package:**/Frameworks/SentrySwift.framework/**         -app
 family:native package:**/Frameworks/Sentry.framework/**              -app
 
 # Sentry internal functions in Cocoa SDK
-family:native function:kscm_*                                        -app -group
-family:native function:kscrash_*                                     -app -group
-family:native function:"?[[]KSCrash *"                              -app -group
-family:native function:"?[[]SentryClient *"                         -app -group
-family:native function:"?[[]SentryHub *"                            -app -group
-family:native function:"?[[]RNSentry *"                             -app -group
-family:native function:"?[[]SentrySDK *"                             -app -group
+# Needed for versions < sentry-cocoa 6.3.0 and static linking. Since sentry-cocoa 6.3.0 we
+# only mark the main executable as inApp and not as well dynamic libs inside the application bundle.
```

Comment:
This is a bit confusing:

> only mark the main executable as inApp and not as well dynamic libs inside the application 

Do you mean we mark only the main executable as `inApp` and nothing else. including dynamic libraries used by the application?

Reply:
Is it clearer now?

Reply:
for me it is